packages/client/src/app/pages/forms/edit/builder/tabs/layout-editor/field-layout/page-tabs/tab/tab.tsx

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
-import { Page } from '../../../../../types/layout';
+import { unfocus } from '@ff-client/app/pages/forms/edit/store/slices/context';
+import { useAppDispatch } from '@ff-client/app/pages/forms/edit/store/store';
+
+import type { Page } from '../../../../../types/layout';
```

Comment:
Good opportunity here to setup aliases and avoid this horrible `../../../../../` imports and do `@ff-types/layout`. You get the gist...

One for the future though.